opendbc/car/ford/radar_interface.py

```diff
@@ -68,6 +71,10 @@ def update(self, can_strings):
       self._update_delphi_esr()
     elif self.radar == RADAR.DELPHI_MRR:
       self._update_delphi_mrr()
+      print('pts', len(self.pts), self.rcp.vl['MRR_Header_InformationDetections']['CAN_NUMBER_OF_DET'])
```

Comment:
`CAN_NUMBER_OF_DET` was always lower by 3-6 times from our number of points, now they match exactly!

It was originally thought that `CAN_SCAN_INDEX_2LSB_xx` was a sort of multiplexing signal, but it makes more sense that along with the header message `MRR_Header_InformationDetections` it's used to ensure that the latest updates are kept up to sync, the LSB always matches the counter in the header message when waiting on the header message now.